### PR TITLE
feat: Cart

### PR DESCRIPTION
**Pull request by Shivanshsharma01**
image

### CODE REVIEW

**Review by Hamdrive — COMMENTED** [review]
Good work done, will suggest to take care of code formatting
Rest all code looks fine.

File: Page/cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        <nav class="nav-container">
+            <div class="nav-logo"><a href="index.html">Dope Kicks <i class="fa-solid fa-boot"></i></a></div>
+            <input type="search" placeholder=" Search items here" class="nav-input">
+            <button class="nav-btn"><a href="#">Home</a></button>
+            <a class="cart-icon" href="/page/cart/cart.html"><i class="fa-solid fa-cart-shopping-fast"></i></a>
+            <a class="cart-icon" href="/page/logout/logout.html"><i class="fa-solid fa-right-from-bracket"></i></a>
+
+        </nav>
+
+        <main>
```
Comment (junaidshaikh-js) [inline Page/cart/cart.html:28]: `header` element should end here. 

Don't nest main inside header

**Review by junaidshaikh-js — COMMENTED** [review]
Clean code! Good use of semantic HTML elements. 

Suggested one change.

File: Page/cart/cart.html (hunk 1 above)
Comment (Shivanshsharma01) [reply Page/cart/cart.html:28]: done

**Pull request merged**